app/views/spree/shared/_login.html.erb

```diff
@@ -16,4 +16,20 @@
       <%= f.submit Spree.t(:login), class: 'btn btn-lg btn-success btn-block', tabindex: 3 %>
     </div>
   </fieldset>
-<% end %>
\ No newline at end of file
+<% end %> -->
+<%= form_for Spree::User.new, as: :spree_user, url: spree.create_new_session_path do |f| %>
+  <%= render :partial => 'spree/shared/sign_in_header' %>
+  <fieldset id="password-credentials">
+    <div class="form-group mb-20">
+      <%= f.email_field :email, class: 'form-control username-control', tabindex: 1, placeholder: Spree.t(:email) %>
```

Comment:
Add this text: 'Email Address*' against this key: Spree.t(:email). Please dnt remove the key .Add the text inside local file.